BoxContentSDK/BoxContentSDK/Cache/BOXRequestCache.m

```diff
@@ -135,4 +136,15 @@ - (void)clearForLogout
     }
 }
 
+- (BOOL)shouldRemoveCachedResponseForRequestError:(NSError *)error
```

Comment:
Perhaps this belongs in BOXRequest so that it can be extended for different types of requests?